modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -57,82 +57,149 @@ podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal]
 ----
-podman push --remove-signatures registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version} default-route-openshift-image-registry.<hostname>/<project_name>/rhdh-hub-rhel9:{product-version}
+oc login -u <user> -p <password> https://api.<hostname>:6443
 ----
+
+. Log in to the {ocp-short} image registry using the `podman` command line tool by running the following command:
 +
 [source,terminal]
 ----
-podman push --remove-signatures registry.redhat.io/rhel9/postgresql-15:latest default-route-openshift-image-registry.<hostname>/<project_name>/postgresql-15:latest
+podman login -u kubeadmin -p $(oc whoami -t) default-route-openshift-image-registry.<hostname>
 ----
 +
-For more information about pushing images directly to the {ocp-short} image registry, see https://access.redhat.com/solutions/6959306[How do I push an Image directly into the OpenShift 4 registry].
-+
-[IMPORTANT]
+[NOTE]
 ====
-If an x509 error occurs, verify that you have link:https://access.redhat.com/solutions/6088891[installed the CA certificate used for {ocp-short} routes on your system].
+You can run the following commands to get the full host name of the {ocp-short} image registry, and then use the host name in a command to log in:
+
+[source,terminal]
+----
+REGISTRY_HOST=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
+----
+
+[source,terminal]
+----
+podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
+----
 ====
 
-. Use the following command to verify that both images are present in the internal {ocp-short} registry:
+. Log in to the `registry.redhat.io` in `podman` by running the following command:
 +
 [source,terminal]
 ----
-oc get imagestream -n {my-product-namespace}
+podman login registry.redhat.io
 ----
-
-. Enable local image lookup for both images by running the following commands:
++
+. Create an `ImageSetConfiguration` file to specify the resources that you want to mirror. For example:
++
+[source,terminal]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  local:
+    path: ./mirror-output
+mirror:
+  helm:
+    repositories:
+      - name: openshift-charts
+        url: https://charts.openshift.io
+        charts:
+          - name: redhat-developer-hub
+            version: "{product-version}"
+----
+. Mirror the resources specified in the`ImageSetConfiguration` file by running the following command:
 +
 [source,terminal]
 ----
-oc set image-lookup postgresql-15
+oc-mirror --config=mirror-config.yaml file://mirror-archive
 ----
+. Transfer the generated `.tar` archive file (for example, `mirror++_++seq1++_++000000.tar`) to the air-gapped environment using `scp` or another file transfer tool. For example:
 +
 [source,terminal]
 ----
-oc set image-lookup  rhdh-hub-rhel9
+scp ./mirror-archive/mirror_seq1_000000.tar /tmp
```

Comment:
```suggestion
scp ./mirror-archive/mirror_seq1_000000.tar [user@]host:/tmp/mirror_seq1_000000.tar
```

If we want to keep this example, I think it should highlight that this is a remote transfer.